xds/src/test/java/io/grpc/xds/CdsLoadBalancer2Test.java

```diff
@@ -586,7 +586,7 @@ private final class FakeLoadBalancerProvider extends LoadBalancerProvider {
 
     @Override
     public LoadBalancer newLoadBalancer(Helper helper) {
-      FakeLoadBalancer balancer = new FakeLoadBalancer(policyName, helper);
+      FakeLoadBalancer balancer = new FakeLoadBalancer(policyName);
```

Comment:
@sergiitk removing unused `deliverSubchannelState` caused a bit of a chain reaction - that was the only place where `helper` was referenced.